cli/pkg/release/gbm.go

```diff
@@ -2,6 +2,9 @@ package release
 
 import (
 	"fmt"
+	"io"
+	"net/http"
+	"os"
```

Comment:
These imports are mostly scaffolding at this point just to keep the linter from throwing errors while we continue porting.